interop-tests/ping-version.json

```diff
@@ -0,0 +1,18 @@
+{
+  "id": "rust-libp2p-head",
+  "containerImageID": "rust-libp2p-head",
+  "transports": [
+    "ws",
+    "tcp",
+    "quic-v1",
+    "webrtc"
+  ],
+  "secureChannels": [
+    "tls",
+    "noise"
+  ],
+  "muxers": [
+    "mplex",
+    "yamux"
+  ]
+}
```

Comment:
We don't need this file here (for v050), right? (I know I added it)

Reply:
Not for this test itself, but we need it in master in general. It does nothing here.